.github/workflows/service-extensions-publish.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Service Extensions Callout images packages
+
+on:
+  push:
+    tags:
+      - 'v*.*'
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Docker image tag to use for the package'
+        required: true
+        default: 'dev'
+      commit_sha:
+        description: 'Commit SHA to checkout'
+        required: true
+      set_as_latest:
+        description: 'Set the tag as latest'
+        required: false
+        default: 'false'
+
+permissions:
+  contents: read
+  packages: write
+
+env:
+  TAG_NAME: ${{ github.ref_name || github.event.inputs.tag_name }}
+  REF_NAME: ${{ github.ref || github.event.inputs.commit_sha }}
+  COMMIT_SHA: ${{ github.sha || github.event.inputs.commit_sha }}
+  PUSH_LATEST: ${{ github.event.inputs.set_as_latest || 'true' }}
+
+jobs:
+  publish-service-extensions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: ${{ env.REF_NAME }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # V3.2.0
```

Comment:
Is there a need for QEMU here ? Are you building for arm64 ?

Reply:
Yep, arm64 is supported with 

```diff
+ platforms: linux/amd64,linux/arm64
```

Reply:
I'll seperate the build process using arm64 runners

Reply:
It's done in 673ec27 😄